help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -0,0 +1,153 @@
+
+- The real-time destination system will attempt to send the events with a p95 of \<10 mins after a Profile segmentation/Update event. This means that 95% of the time, profile segment updates will be delivered to Magnite Streaming in under 10 min. The actual receipt and processing of the events within Magnite Streaming depends on the shared data volume.
+
+- Post-ingest, segments are expected to appear in Magnite Streaming within a few minutes and can be applied to a deal. The ad server may take up to an hour to acknowledge targeting updates.
+
+- Audiences shared to Magnite Streaming using the real-time destination will also need to be shared using the Magnite [Streaming Daily destination](magnite-batch.md).
+
+For more information on whether this Real-time destination is right for you, please contact your Magnite Streaming account representative.
+
+## Prerequisites
+
+To use the Magnite destinations in the Adobe Experience Platform, users must first have a Magnite Streaming account. If you have a Magnite Streaming account, please reach out to your account manager to gain access to the Magnite destinations.
+
+## Supported Identities
+
+Magnite Streaming destinations can receive the following identifier sources from the Adobe CDP. Please note all of the following identity sources can map to the Magnite device_id target identifier:
+
+| Identity Source             | Description                                                                                      | Consideration                                                                         |
+|:--------------------------- |:------------------------------------------------------------------------------------------------ |:------------------------------------------------------------------------------------- |
+| TV Device IDs               | Identifiers for advertising across Over-the-top (OTT)/ Connected TV (CTV) devices and platforms. | These IDs may be referenced as a custom identity source for the user.                 |
+| MAIDs                       | Google Advertising ID (GAID) or Apple ID for Advertisers (IDFA).                                 | These IDs are generally captured in the GAID and IDFA source namespaces for the user. |
+| Publisher Provided User Ids | This is generally a first-party, cross-device identifier that is included in the ad request.     | This may be referenced as a custom identity source for the user.                      |
+
+## Setting Up The Destinations
+
+Once your destination usage has been approved and Magnite has shared your credentials, please follow the below steps to authenticate, map, and share data.
+
+### **Authenticate to destination**
+
+Locate the Magnite Streaming destinations in the Adobe Experience catalog. Click the additional options button (\...) and then configure the destination.
```

Comment:
No need for this first sentence. Explaining the catalog entry is already done in the https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination guide, which is referenced in the template, and you removed.

Reply:
No need for this part "Locate the Magnite Streaming destinations in the Adobe Experience catalog. Click the additional options button (\...) and then configure the destination."